conf/routes

```diff
@@ -6,13 +6,14 @@
 GET     /                                   controllers.HomeController.index
 GET     /reset                              controllers.AdminController.resetData
 GET     /projects                           controllers.ProjectsController.getProjects(limit: Int ?=5)
-POST    /projects                           controllers.ProjectsController.addProject
-GET     /projects/:projectId                controllers.ProjectsController.getProjectById(projectId: String)
+POST    /projects                           controllers.ProjectsController.addProject()
```

Comment:
What's the difference in using brackets or not at the method references? Please use uniformly.

Reply:
Methods that follow JavaBean naming contract for accessors are expected to have no side effects. The recommended convention is to use a parameterless method whenever there are no parameters and the method have no side effect. This convention supports the uniform access principle, which says that client code should not be affected by a decision to implement an attribute as a field or method. The problem is that Java does not implement the uniform access principle. To bridge that gap, Scala allows you to leave off the empty parentheses on an invocation of function that takes no arguments.